Tests/test_phyml_tool.py

```diff
@@ -43,6 +43,8 @@ class AppTests(unittest.TestCase):
 
     def test_phyml(self):
         """Run PhyML using the wrapper."""
+        if not os.getenv("PHYMLCPUS"):
+            os.putenv("PHYMLCPUS", "1")
```

Comment:
Took me a while to see this said "PHYML CPUs = 1", meaning single threaded. Maybe add a one line comment here explaining why you want it single threaded if not configured otherwise?

Reply:
Good point, rereading myself the day after, I find it unclear as well.  I push a descriptive comment now.  Thanks for your review.